resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+################
+# Usage: controller.sh <os>
+# The controller script does all of the jar compilation
+# and setup needed to build packages for <os>
+# The os definition is fairly flexible using string matching
+# and currently supports el6, el7, sles11, sles12, debian jessie,
+# debian stretch, ubuntu trusty, ubuntu xenial, and ubuntu yakkety
+################
+build_os=$1
+
+if [ ! -d base ]; then
+        DESTDIR=base bash install.sh install_redhat
+fi
+if [ ! -d systemd_el ]; then
+        cp -r base systemd_el
+        DESTDIR=systemd_el bash install.sh systemd_redhat
+fi
+if [ ! -d old_el ]; then
+        cp -r base old_el
+        DESTDIR=old_el bash install.sh sysv_init_redhat
+fi
+if [ ! -d old_sles ]; then
+        cp -r base old_sles
+        DESTDIR=old_sles bash install.sh sysv_init_suse
+fi
+
+# things are only different if we have docs, deb docs get
+# installed in an unversioned folder but rpm docs get installed
+# in a versioned folder.
```

Comment:
Yeah, debs are in unversioned dirs and rpms at least as far back as el6 use versioned dirs which is handled in https://github.com/puppetlabs/ezbake/pull/429